data/data-files/server-side-eval/errors-bad-attribute-reference.yml

```diff
@@ -0,0 +1,55 @@
+---
+name: bad attribute reference errors - <NAME>
+
+parameters:
+  - NAME: clause with no attribute
+    CLAUSE: { "op": "in", "values": [""] }
+    CONTEXT: { kind: "user", key: "user-key" }
+  
+  - NAME: empty path component
+    CLAUSE: { contextKind: "user", "attribute": "///", "op": "in", "values": [""] }
+    CONTEXT: { kind: "user", key: "user-key" }
```

Comment:
Could we also have one with `"attribute" : null`, to be sure we treat that the same as no attribute?  

In the Rust SDK, for example, our serialization library sees this differently than omission of the attribute, so we need to be sure to treat it the same.